.github/mergify.yml

```diff
@@ -169,3 +169,8 @@ queue_rules:
     update_bot_account: Mikolaj
     merge_method: squash
     update_method: merge
+
+defaults:
+  actions:
+    backport:
+      title: "Backport of #{{ number }}: {{ title }}"
```

Comment:
Any chance we can skip "of "? It can make a difference for me (I read a lot of these via emails on a smartphone, and the titles get cut in preview for me). Without the "of" you can read "backport" as a verb, and then it becomes somewhat grammatically correct... (I also love verbs over nouns in general)

Reply:
You said you wanted something you could reliably match in an email, so I did that. I can change it, though.

Reply:
Thank you!

Maybe it's bad to overfit it to my personal workflows but I hope it does no harm to others. But let me know if I'm mistaken.

I don't believe, with my setup, I can match on exact phrases like "Backport of" anyway. And in any case, I don't want automatic filtering, I just want a short herald in the front to ease manual triage of my emails from this repo. The current version should work better (I hope).

Reply:
BTW, I would disagree with your noun vs. verb issue, specifically because it _is_ a verb: it's saying to backport something, not that it _has been_ backported.